miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -1081,8 +1081,8 @@ end
 #!
 #! Where:
 #! - block_height_delta is the desired expiration time delta (1 to 0xFFFF).
-export.update_expiration_block_num
-    exec.tx::update_expiration_block_num
+export.update_expiration_delta
+    exec.tx::update_expiration_delta
 end
```

Comment:
I would keep the name as it was - but I would update the doc comments to be similar to the one in `kernel::tx::update_expiration_block_num`.

The current doc comments would work well for the new procedure though.